src/main/java/wooteco/subway/service/SpringLineService.java

```diff
@@ -0,0 +1,43 @@
+package wooteco.subway.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import wooteco.subway.dao.LineDao;
+import wooteco.subway.domain.Line;
+import wooteco.subway.dto.LineRequest;
+
+@Service
+public class SpringLineService implements LineService {
+
+    private final LineDao lineDao;
+
+    public SpringLineService(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    @Override
+    public Line save(LineRequest lineRequest) {
+        Line line = new Line(null, lineRequest.getName(), lineRequest.getColor());
```

Comment:
```suggestion
        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
```
도메인의 편의생성자를 추가해보는건 어떨까요?

Reply:
이 방식이 더 좋겠네요. 수정하겠습니다,